### ISSUE TITLE
sudo apt-add-repository -y ppa:rael-gc/rvm

### ISSUE THREAD

**Issue by Thornybastard** [rvm/ubuntu_rvm#33]
OK, I'm a total noob to Ubuntu/linux. I've been through all the other related issues here and though most of it is Greek to me, I managed to try some of the fixes. None worked and I'm not behind a proxy, I'm using my home connection.

Every time I try: sudo apt-add-repository -y ppa:rael-gc/rvm 

I get:
Cannot add PPA: 'ppa:~rael-gc/ubuntu/rvm'.
ERROR: '~rael-gc' user or team does not exist.

Looks the same as the guy back in June, only I know I'm not behind a proxy, unless there's something I don't know.

I'm using Ubuntu 16.04 on Windows 10, and I ran the update and the upgrade before trying. This is literally my first day as a linux user. Learned Ruby no problem and now wanna get into Rails.

**Comment by raelgc** [comment]
Hi @Thornybastard,

Please, check if any solution from this [topic](https://askubuntu.com/questions/863825/unable-to-add-ppa-in-my-newly-installed-ubuntu-16-04-lts) will help you.

**Comment by Thornybastard** [comment]
No. None of them worked. And after I tried purging the ca-certificates, autoremove, and then reinstalling the ca-certificates, ALL of the connections failed and no ca-certificates were installed.

So then I tried reinstalling software-properties common first and then reinstall the ca-certificates, but that also failed. Could not even install software-properties common.

I've also tried installing keys before trying to get the ppa, but it never fetches the keys either.  It seems like this is some new issue. Since I'm a totla noob I cannot troubleshoot it. I don't know what to look for and I don't know how to access directories.

I think using Cloud 9 is the solution instead of trying to figure out Ubuntu.
